pkg/resource/plugin/plugin.go

```diff
@@ -126,7 +126,7 @@ func newPlugin(ctx *Context, bin string, prefix string, args []string) (*plugin,
 			killerr := plug.Proc.Kill()
 			contract.IgnoreError(killerr) // we are ignoring because the readerr trumps it.
 			if port == "" {
-				return nil, errors.Wrapf(readerr, "could not read plugin [%v] stdout", bin)
+				return nil, errors.Wrapf(readerr, "error attempting to load plugin [%v] stdout", bin)
```

Comment:
The new text doesn't really work with the `stdout` at the end of the sentence.  I also honestly don't follow how/why this is different than the message two lines lower.  

Why exactly are we changing this?